setup.py

```diff
@@ -37,7 +37,7 @@
         'SQLAlchemy',
     ],
     package_data={'': ['*.json', '*.gz', '*.csv']},
-    setup_requires=['pytest-runner'],
+    setup_requires=[],
```

Comment:
This is probably the one controversial change in this PR... [Here's my rationale](https://github.com/openeemeter/eemeter/pull/125/commits/8b644b99e2a17c6aeb083f2e46a0580e351bd53c), but I'm open to feedback.